_posts/2020-02-29-f12.markdown

```diff
@@ -9,15 +9,15 @@ tags:
   - Coding
 ---
 
-I already had to create bootable floppy images in some of my projects .
+I've already had to create bootable floppy images in some of my projects.
 This process has always been a hassle, especially automating it.
-So I decided to write my own all-in-one solution to create FAT12 formatted
+So I've decided to write my own all-in-one solution to create FAT12 formatted
 images - f12.
 f12 is a program written in C, that provides various functions to handle FAT12
 formatted images, for example moving files to the image, dumping files from the
 image and listing contents of directories on the image.
-But the most import feature of all for me is creating new images with a set
-files and installing a bootloader in one command, but let's start at the
+But the most import feature of all for myself is creating new images with a set
+file and installing a bootloader in one command, but let's start at the
```

Comment:
```suggestion
of files and installing a bootloader in one command, but let's start at the
```

Reply:
Hier rede ich von mehreren Dateien, die ich gerne auf das Image schreiben würde.

Reply:
Gut dann: But the most important (Grad noch ne fehler gesehen lul) feature of all for myself, is creating new images with set files and installing...

Reply:
Stimmt, `important` habe ich auch übersehen